src/workerd/jsg/jsvalue.c++

```diff
@@ -300,7 +301,40 @@ size_t JsString::utf8Length(jsg::Lock& js) const {
 }
 
 kj::String JsString::toString(jsg::Lock& js) const {
-  return kj::str(inner);
+  return js.accountedKjString(kj::str(inner));
```

Comment:
I'm concerned that the accounting adds considerable overhead for small strings, do we really want to do it automatically by default? Maybe this should apply only to large strings?

Reply:
This PR does not change any of the accounting. If we are seeing a negative impact from the accounting that was added then we can/should address that in a separate PR.

Reply:
We haven't measured in any meaningful way yet but that's something that can be looked at separately.